### PR TITLE
Fix bad link to device-initialization

### PR DESCRIPTION
**Pull request by nealmcb**
I'm not positive how to specify the link, just a guess. Let's see if it works.

### CODE REVIEW

File: piv/yubikey-piv-manager.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ The [Yubikey PIV Manager](https://developers.yubico.com/yubikey-piv-manager) is
 
 The command-line utility is more powerful (e.g. allows overriding pin and touch default slot policies) but it also more prone to user errors. It's very easy to overwrite an existing key, for instance.
 
-After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](#device-initialization).
+After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](device-initialization).
```
Comment (ruimarinho) [inline piv/yubikey-piv-manager.md]: I think it needs to be `../device-initialization/README.md`. Can you do that change for me?

File: piv/yubikey-piv-manager.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ The [Yubikey PIV Manager](https://developers.yubico.com/yubikey-piv-manager) is
 
 The command-line utility is more powerful (e.g. allows overriding pin and touch default slot policies) but it also more prone to user errors. It's very easy to overwrite an existing key, for instance.
 
-After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](device-initialization).
+After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](../device-initialization//README.md).
```
Comment (ruimarinho) [inline piv/yubikey-piv-manager.md]: Oops, double slash -- can you squash your commits after that?

Comment (nealmcb) [reply piv/yubikey-piv-manager.md]: Oops!! Done. Thanks again for your patience.

Comment (nealmcb) [reply piv/yubikey-piv-manager.md]: And tested!
I suggest a .gitignore for _book and node_modules

**Comment by nealmcb** [comment]
OK, fixed, squashed and tested!
I suggest a .gitignore for _book and node_modules

**Comment by ruimarinho** [comment]
Perfect :) thank you so much! If you still have time for the `.gitignore` piece, I would be willing to merge it!

**Pull request merged**